client/editSchedule.js

```diff
@@ -193,9 +241,11 @@ setupCompetitionCalendar = function(template, $calendarDiv, $editModal) {
       },
     });
   });
+
   template.autorun(function() {
     var data = Template.currentData();
-    var rounds = getRoundsWithSchedule(data.competitionId);
+    // calendarRounds = getRoundsWithSchedule(data.competitionId);
```

Comment:
Vestigial line of code, please delete. Can we delete the old getRoundsWithSchedule() method too?